src/components/ShareButton.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import classnames from 'classnames'
+
+const ShareButton = ({ label, disabled, onShare, className }) => (
+  <button
+    role='button'
+    disabled={disabled}
+    className={classnames(...className, 'coz-btn', 'coz-btn--secondary', 'coz-btn--share')}
```

Comment:
Just for my personal knowledge, what does `…className` means? I would have expect just `className` but the `…` just intrigues me.